pallets/identity/src/lib.rs

```diff
@@ -519,13 +519,13 @@ decl_module! {
 
             match &claim {
                 Claim::CustomerDueDiligence(..) => Self::base_add_cdd_claim(target, claim, issuer, expiry)?,
-                Claim::InvestorZKProof(_t, _s, cdd_id, _p) => {
+                Claim::InvestorZKProof(..) => {
                     Self::base_add_confidential_scope_claim(
                         target,
                         claim.clone(),
                         issuer,
-                        expiry,
-                        cdd_id.clone())?
+                        expiry
+                    )?
                 },
                 _ => {
                     T::ProtocolFee::charge_fee(ProtocolOp::IdentityAddClaim)?;
```

Comment:
Should we charge a protocol fee for `InvestorZKProof` claim as well?

Reply:
We should charge but it was written like this so I thought this was the business decision.
@adamdossa to confirm.

Reply:
I think we can leave this as it is for now. Since ZKProof claims are "mandatory" it seems a bit harsh to charge fees for them.

Reply:
I'm pretty agnostic. I think for now though this is fine - scope label & CDD claims can be special cased as having no protocol fee.